game.js

```diff
@@ -1,51 +1,83 @@
-const rls = require('readline-sync')
+const rls = ({ keyInYN } = require('readline-sync'));
 
 /**
  * Starts the game by prompting the user if they want to play
  * Calls either gameLoop() or quitGame()
- * 
+ *
  * @returns {undefined}
  */
 const startGame = () => {
-
-}
+	if (rls.keyInYN('Hey want to play?')) {
+		console.log(`I have a random number in mind`);
+		console.log(`It's between 1 and 1000`);
+		console.log(`You have 10 guesses total`);
+		console.log('Want to play again?');
+		console.log("Let's start!");
+		gameLoop();
+	} else {
+		console.log('Have a nice life!');
+		quitGame();
+	}
+};
 
 /**
  * Logs "Goodbye!"
  * Calls process.exit() to quit the game
- * 
+ *
  * @returns {undefined}
  */
 const quitGame = () => {
-
-}
+	console.log('Goodbye!');
+	process.exit();
+};
 
 /**
  * Controls the flow of the game.
  * Should prompt the user to play again once all
  * guesses have been made or correct answer guessed
- * 
+ *
  * @returns {undefined}
  */
-const gameLoop = () => {
-
+const gameLoop = () => {};
+startGame();
+if (rls.keyInYN('Hey want to play?')) {
+} else {
+	quitGame();
 }
 
-
 /***
- * Generates a random number 
+ * Generates a random number
  *
  * @returns {number} - a number between 1 and 1000
  */
 const generateRandomNumber = () => {
```

Comment:
This function is doing way too much. All this function should do is return a number between 1 and 1000. Use google to help you find how to make a random number in that range.

Reply:
Hello Corey'

Thank you for the comment I also agree it is doing way way way way to much ; 
below is what i simplified it to and it work.
const generateRandomNumber = () => {
	let number = (Math.random() * 1000);
	return number;